spec/system/index_spec.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe "Index page", type: :feature do
+
+    it "includes the Smartflix title" do
```

Comment:
The indentations could be better but don't worry about it now. We'll have a ticket for installing Rubocop.